frontend-app/src/pages/payable/ListPayables.tsx

```diff
@@ -0,0 +1,44 @@
+import { useEffect, useState } from "react";
+import { Payable } from "../../types";
+import { axiosInstance as axios } from "../../api";
+import PayableCard from "../../components/PayableCard";
+import { getTokenAndSetHeaders } from "../../helpers";
+
+function ListPayables() {
+  const [payables, setPayables] = useState<Payable[]>();
+
+  useEffect(() => {
+    const config = getTokenAndSetHeaders();
+    const getPaybales = async () => {
+      try {
+        const { data } = await axios.get('/payable', config);
+        setPayables(data);
+
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      } catch (err: any) {
+        alert('Você não está logado.');
+      }
+    }
+    getPaybales();
+  }, []);
```

Comment:
aqui talvez criar um `service` para ir fazendo essas chamadas, isso acaba ajudando quando você for usar a mesma função em outro lugar